src/ui/services/Background.ts

```diff
@@ -209,19 +203,9 @@ class Background {
     }
   }
 
-  async nftInfo(address: string, limit: number = 1000): Promise<AssetDetail[]> {
+  async assetFavorite(assetId: string): Promise<AssetDetail> {
```

Comment:
The return type should be `Promise<void>` now, since it's not returned from background.js and controller